src/Runner/Hook.php

```diff
@@ -360,6 +379,55 @@ public static function getExecMethod(string $type): string
         return $valid[$type];
     }
 
+    /**
+     * Check if the action should be skipped
+     *
+     * @param Config\Action $action
+     * @return bool
+     */
+    private function checkSkipAction(Config\Action $action): bool
+    {
+        if (
+            $this->shouldSkipActions()
+            || $this->cliSkipAction($action)
+            || !$this->doConditionsApply($action->getConditions())
+        ) {
+            $this->io->write('<comment>skipped</comment>', true);
+
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Check if the CLI `action` options indicates the action should be skipped
+     *
+     * @param Config\Action $action
+     * @return bool
+     */
+    private function cliSkipAction(Config\Action $action): bool
+    {
+        /** @var IO\Base $io */
+        $io = $this->io;
```

Comment:
This is for the benefit of PHPStan, since `getOption()` is not on the `Console\IO` interface.